### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'split'

### ISSUE THREAD

**Issue by cyberdee** [bhavesh95863/whitelabel#19]
This happens when I tried the apps on my V13.

![image](https://user-images.githubusercontent.com/19448615/175522662-e8e8f9f6-1fbd-49d9-a125-3e85feaa0cfa.png)

**Comment by elan-k** [comment]
Yes. I'm also getting this error while clicking on the save button.

**Comment by TechAbout** [comment]
Same here.

**Comment by samjaninf** [comment]
Try replacing this line in api.py:
`return frappe.db.get_value("Installed Application",{"app_name":"frappe"},"app_version").split('.')[0]`
```python
        frappe_version = next((item for item in frappe.utils.get_installed_apps_info() if item['app_name'] == 'frappe'), '13.0')
        return frappe_version['version'].split('.')[0]
```

**Comment by Thedigitalcontractor1** [comment]
> Try replacing this line in api.py: `return frappe.db.get_value("Installed Application",{"app_name":"frappe"},"app_version").split('.')[0]`
> 
> ```python
>         frappe_version = next((item for item in frappe.utils.get_installed_apps_info() if item['app_name'] == 'frappe'), '13.0')
>         return frappe_version['version'].split('.')[0]
> ```

make sure to delete spaces in line 2 and replace with tab
